### PR TITLE
Update ReadME

### PR DESCRIPTION
**Pull request by SnowNooDLe**
Split from one ReadMe to several ReadMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,325 @@
...
+
+## Branches
+
+### master_dev
+
+Treat as a playground, once the new feature seems working fine, push to `master_dev` from the working branch so it can be deployed.
+
+### master_ea
+
+Treat as `master`, the app should be as stable as possible.
```
Comment (jasonmotha) [inline README.md:11]: It's real confusing as to the progression of code. A link to a flowchart might be useful.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,325 @@
...
+
+If it is in DEV version, you see two options, **v20p5emp** and **v20p45sim** for **Ensemble** but in EA and PROD, they are not visible.
+
+##### Location
+
+Users can put Lat and Lng within NZ coordinates. Then click **Set** button to get a station, regional map and vs30 map.
+
+##### Site Conditions
+
+By setting the location, users get a default VS30. They can also update VS30 if they want by updating the input field and click **Set VS30**. By clicking the **Use Default**, it goes back to what it was. (The value from API when users set the location)
```
Comment (jasonmotha) [inline README.md:74]: Vs30?

File: README.md (hunk 1 above)
Comment (SnowNooDLe) [reply README.md:11]: I will add a flowchart next week.

**Review by claudio525 — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Review by jasonmotha — APPROVED** [review]
That chart looks really good. And makes it so much clearer.

**Pull request merged**